server.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra'
+require 'multi_json'
+require 'net/http'
+require 'uri'
+
+configure do
+  set :port, 5000
+end
+
+get '/' do
+  send_file 'index.html'
+end
+
+post '/user/verify' do
+  data = MultiJson.load(request.body.read)
+
+  url = URI.parse(data['apiUrl'])
+  req = Net::HTTP::Get.new(url.path)
+  req['Authorization'] = data['authHeader']
+
+  http = Net::HTTP.new(url.host, url.port)
+  http.use_ssl = true if url.scheme == 'https'
+  res  = http.request(req)
+
+  halt res.code.to_i, 'Error! Request failed.' unless res.code == '200'
+
+  MultiJson.dump(res.body)
```

Comment:
So the Python version actually does the following:

```python
return jsonify(json.loads(response.content))
```
https://github.com/AnSavvides/react-digits/blob/master/server.py#L25

This is parsing the JSON via `json.loads` and re-serializing to JSON again via the call the `jsonify`. This seemed odd to me so I just made the Ruby code always serialize to JSON but didn't unnecessarily parse the response here for no reason.

Should we just return the respond body in both Python and Ruby?